htdocs/core/class/commondocgenerator.class.php

```diff
@@ -1082,7 +1082,8 @@ public function printStdColumnContent($pdf, &$curY, $colKey, $columnText = '')
         $parameters = array(
             'curY' => &$curY,
             'columnText' => $columnText,
-            'colKey' => $colKey
+            'colKey' => $colKey,
+            'pdf' => $pdf,
```

Comment:
Is it possible to change this line into
'pdf' => &$pdf,

Reply:
If you want to but object are always passed by refrence.